spring-cloud-aws-messaging/src/main/java/org/springframework/cloud/aws/messaging/listener/QueueMessageHandler.java

```diff
@@ -73,15 +74,18 @@
 	static final String LOGICAL_RESOURCE_ID = "LogicalResourceId";
 	static final String ACKNOWLEDGMENT = "Acknowledgment";
 	static final String VISIBILITY = "Visibility";
+	private final   SqsMessageDeletionPolicy defaultSqsMessageDeletionPolicy;
```

Comment:
remove extra space. Also, can be renamed to `sqsMessageDeletionPolicy `

Reply:
Done